src/main/java/com/exadel/sober/services/AddictionService.java

```diff
@@ -0,0 +1,28 @@
+package com.exadel.sober.services;
+
+import com.exadel.sober.exceptions.CannotAddAddictionException;
+import com.exadel.sober.models.Addiction;
+import com.exadel.sober.repositories.AddictionRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class AddictionService {
+    private AddictionRepository addictionRepository;
+
+    public AddictionService(AddictionRepository addictionRepository) {
+        this.addictionRepository = addictionRepository;
+    }
+    public List<Addiction> findAllAddiction() {
+        return addictionRepository.findAll();
```

Comment:
pay attention to the case if user does not have addictions yet. What will be the result of addictionRepository.findAll()? If null or exception - you must update your code and return empty list instead exception or null.

Reply:
rechecked it, in case if there isn't any addiction, returning empty array - work OK